.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
shouldn't be in the commit. Maybe add a second commit to add `.pytest_cache/` to `.gitignore`?